src/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "src",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
add a script dev to run index.ts using nodemon,
have a look at [ts-node](https://www.npmjs.com/package/ts-node)